flake.nix

```diff
@@ -18,28 +18,18 @@
   }: let
     inherit (nixpkgs) lib;
 
-    version = "0.23.0"; # x-release-please-version
-
     # Helper to provide system-specific attributes
     forAllSystems = f:
       lib.genAttrs systems (system: let
         pkgs = nixpkgs.legacyPackages.${system};
         zpkgs = zigpkgs.legacyPackages.${system};
-        beamPackages = pkgs.beam_minimal.packages.erlang_27;
-        otp = (pkgs.beam.packagesWith beamPackages.erlang).extend (final: prev: {
-          elixir_1_17 = prev.elixir_1_16.override {
-            rev = "v1.17.0";
-            # You can discover this using Trust On First Use by filling in `lib.fakeHash`
-            sha256 = "sha256-RBylCfD+aCsvCqWUIvqXi3izNqqQoNfQNnQiZxz0Igg=";
-            version = "1.17.0";
-          };
-
+        beamPackages = pkgs.beam_minimal.packages.erlang_27.extend (final: prev: {
```

Comment:
i don't think we need to extend this anymore

Reply:
I left this in because two erlangs have to be compiled for this config, both 26 and 27. https://github.com/NixOS/nixpkgs/issues/322168

I'm happy to remove it, but anybody building will have to wait on the slow rebar tests to run.

Reply:
I think that happens anyway, when i pulled down the branch and my direnv starting booting up the nix develop shell, it compiled otp 26 and rebar

Reply:
Yeah we can't avoid compiling for now, but running the *tests* for rebar add significant time to the build. I'll pull it out since most users will be using the cache anyway.

Reply:
Does extending it remove the tests?

Reply:
Yes. It's effectively an override for these nested packages allowing the tests to be disabled.

Reply:
oh, is that what the `doCheck = false` is doing? I had copied that from an example previously and didn't pay it much attention.

Reply:
Yes :)